lib/bn.js

```diff
@@ -885,13 +885,212 @@ BN.prototype._bigMulTo = function _bigMulTo(num, out) {
 
 BN.prototype.mulTo = function mulTo(num, out) {
   var res;
-  if (this.length + num.length < 63)
+  var len = this.length + num.length;
+  if (len < 63)
     res = this._smallMulTo(num, out);
-  else
+  else if (len < 1024)
     res = this._bigMulTo(num, out);
+  else
+    res = this._bigMulToF(num, out);
   return res;
 };
 
+
+// Private
+
+function FFTM(x, y) {
+  this.x = x;
+  this.y = y;
+}
+
+FFTM.prototype.makeRBT = function makeRBT(N) {
+  var t = new Array(N);
+  var l = (Math.log(N) / Math.log(2)) | 0;
+  for (var i = 0; i < N; i++) {
+    t[i] = this.revbin(i, l, N);
+  }
+
+  return t;
+};
+
+FFTM.prototype.revbin = function revbin(x, l, N) {
```

Comment:
It'd help a lot if there would be a comments describing what's actually happening there :)

For example, this function reverses the bits in a word of length `l`. Also, it could be `revBin` since we are using camel case ;)